### ISSUE TITLE
[A11Y] Interactive time tooltip are not accessible

### ISSUE THREAD

**Issue by fsologureng** [go-gitea/gitea#23645]
### Description

With the PR [23526](https://github.com/go-gitea/gitea/pull/23526) the time tooltips began to be interactive, which means that the tippy module enable aria attributes as `aria-expanded`. However this attribute is invalid in the context of the actual estructure of the tooltip caller (the span container) which doesn't use labelledby to link the tooltip and haven't a role neither.

Description of tooltip role in MDN https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role
Description of tooltip role in current WAI-ARIA W3C Recommendation https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#tooltip

### Gitea Version

1.19

### Can you reproduce the bug on the Gitea demo site?

Yes

### Screenshots

![tippy function](https://user-images.githubusercontent.com/5055343/227049019-575bdcfb-5710-4eac-a449-9e1ac45304e4.png)
![AxeCore DevTools critical issue](https://user-images.githubusercontent.com/5055343/227050033-31e05b73-310d-4fba-b2ad-c7231f06e5a3.png)

### How are you running Gitea?

Locally and tested in https://try.gitea.io

### Database

None

**Comment by silverwind** [comment]
Tippy has some [aria options](https://atomiks.github.io/tippyjs/v6/all-props/#aria), what should we set instead for such interactive tooltips?

**Comment by fsologureng** [comment]
> Tippy has some [aria options](https://atomiks.github.io/tippyjs/v6/all-props/#aria), what should we set instead for such interactive tooltips?

Thank you for your guidance to the manual :+1:
